_posts/2017-09-01-sept-01.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: "This Week In Erlang Sept 1"
+description: ""
+date: 2017-09-01
+tags: [erlang, news]
+comments: false
+share: true
+---
+
+Hello and welcome to "This week in Erlang"! This week bunch of interesting happened, and most importantly there is a very important announcement! Get excited.
+
+### Community Announcement
+As a community, we want to make the BEAM community better. This requires us to know who we are, what we do, what we like and what we don't like. Today, thanks to *Mariano Guerra*, we are launching the first in what hopefully becomes a yearly survey on the State of the BEAM, please fill the survey and share it widely so we get a good representation of the community.
+
+http://bit.ly/state-of-beam-2017
+
+### Articles and Blog posts
+ * Mariano (@warianoguerra) has started a new series of tutorials on using riak_ensemble and multi-paxos protocol. Very easy to follow and full of interesting insights: <http://marianoguerra.org/posts/multi-paxos-with-riak_ensemble-part-1/>
+
+ * Fred Hebert (@mononcqc) has published a small guide on ways to make SSL and Erlang safer, if you are into that kind of stuff be sure to take a look: <https://gist.github.com/ferd/af9abf6b3600d2d7f08dba58fdfb514a>
+
+ * Kenji Rikitake (@jj1bdx) has started a discussion on how the `rand` module differs from OTP 19 to OTP 20, <https://groups.google.com/forum/#!topic/erlang-programming/Icw0pcIsnIY>
+
+ * On 25th of August an online meeting was held to discuss the future of riak, this document was available online: https://slack-files.com/T6PE2GNC8-F6U0FEFKP-b5db1a9fcd
+ 
+### Library updates
+ * Benoît Chesneau (@benoitc) has released a wrapper around the embedded DB RocksDB, it is very interesting: <https://gitlab.com/barrel-db/erlang-rocksdb/blob/master/CHANGELOG>
```

Comment:
I found 5 misspelled words in this sentence: 
- <https
- RocksDB
- Chesneau
	- *Suggested Replacements:* cheneau
- Benoît
- *
	- *Suggested Replacements:* , k, p, u, e, z, j, o, t, d, y, i, n, s, c, x, h, m, r, b, w, g, l, q, a, v, f